From 857f90602be065215fe6f5042eb3379bce0cdf86 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Sat, 24 May 2014 17:07:07 +0800
Subject: [PATCH] pyanaconda/packages.py: run pwconv

Run pwconv to convert to use shadow as the passwd

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 pyanaconda/packages.py | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/pyanaconda/packages.py b/pyanaconda/packages.py
index 038e809..dd7ebab 100644
--- a/pyanaconda/packages.py
+++ b/pyanaconda/packages.py
@@ -47,6 +47,13 @@ import gettext
 _ = lambda x: gettext.ldgettext("anaconda", x)
 
 def doPostAction(anaconda):
+    # Convert to shadow passwd
+    rc = iutil.execWithRedirect("pwconv", [],
+            stdout = "/dev/tty5",
+            stderr = "/dev/tty5",
+            root = anaconda.rootPath)
+
+
     anaconda.instClass.postAction(anaconda)
 
 def firstbootConfiguration(anaconda):
-- 
1.8.3.1

